lib/rspec/its.rb

```diff
@@ -1,6 +1,14 @@
 require 'rspec/its/version'
 require 'rspec/core'
 
+RSpec.deprecate <<EOS
+rspec-its deprecates calling _send_ in favour of _public_send_ internally.
+This change will be introduced with version 2.0.
+Maybe you're testing private methods in your test suite without your intention.
+If you need more information about which test is affected,
+set `config.its_raise_errors_for_private_method_calling = true`
+EOS
+
```

Comment:
This will be quite annoying. I would expect to see a deprecation message when I'm testing a private method and there's really a difference between using `send` and `public_send`.

Please also check usages of `deprecate`, its interface is not too friendly, but surely you'll find an example that would fit this usage scenario.